tests/boot/grub_test_snapshot.pm

```diff
@@ -16,14 +16,24 @@ use base "basetest";
 use testapi;
 
 sub run() {
-    assert_screen "grub2";
+    if (get_var('ROLLBACK_AFTER_MIGRATION')) {
+        # set BOOT_TO_SNAPSHOT here rather than in main.pm to prevent conflict with existing snapshot tests
+        set_var('BOOT_TO_SNAPSHOT', 1);
+        select_console 'root-console';
+        wait_still_screen 2;
```

Comment:
that should not be necessary anymore after the implicit assert_screen in select_console now